process/customer-success.md

```diff
@@ -0,0 +1,409 @@
+
+The day to day activities of Tier 1 personnel include interacting with users, recording the issues as they occur, identifying opportunities to grow customer usage, and providing feedback to product management to prioritize product, solution and partnership development.  
+
+When recording issues, Tier 1 personnel triage incoming tickets into the following: 
+
+* Sev 0 : Data is lost or compromised. Nodes or Pods are getting crashed.
+* Sev 1 : Data is not accessible 
+* Sev 2 : Operational Complexity 
+* Sev 3 : New Features or Enhancements to existing features
+
+Tier 2 and Tier 3 are engaged in case of Sev 0 and Sev 1 immediately for engineering debugging and/or hot patch application. Tier 1 plays an important role in reducing the volume of tickets requiring Tier 2 and Tier 3.  When a ticket reaches this level it means our product simply is not working as needed - users are unable to self sustain, have uncovered a bug or other operational issues, and may be at risk of not becoming a reference.  
+
+Product management and engineering are accountable for making sure, when Tier 2 and Tier 3 tickets are in progress, the entire organization need to be working on them to resolve them ASAP both to help particular customers and to insure that we improve the product, solutions, and partnerships and even marketing so such issues are avoided in the future.
+
+OpenEBS products are available in Community and Enterprise editions. The Community Edition is targeted towards tech savvy DevOps personnel who would like to try out and contribute back to the product. Taking the community edition to production requires DevOps to write their own tooling to help with alerting and troubleshooting. 
+
+Enterprise Edition lags behind Community Edition in terms of releases/features, but is bundled with MayaOnline and required support tools that greatly simplify the tasks performed by Customers. The Enterprise Edition involves qualifying the product with the supported platforms using automated e2e via Litmus.
```

Comment:
The scope of enterprise edition now expands to data management service, chaos service etc. Update this section with the latest details.